Navidux/Sources/Navidux/alert/AlertFactory.swift

```diff
@@ -1,9 +1,13 @@
 import UIKit
 
-public final class AlertFactory {
-    func createAlert(configuration : AlertConfiguration) -> AlertScreen {
+public protocol AlertFactory {
+    func createAlert(configuration : AlertConfiguration) -> AlertScreen
```

Comment:
`func createAlert(configuration: AlertConfiguration) -> AlertScreen`

It should be without spacing in argument by r_m_r code style